nnvm/python/nnvm/frontend/pytorch/base.py

```diff
@@ -0,0 +1,119 @@
+r'''Basic classes for PyTorch operators and graphs'''
+from abc import ABC, abstractmethod
+from collections import OrderedDict
+from nnvm.symbol import Variable
+
+
+def _unique_name(node):
+    return next(node.outputs()).uniqueName()
+
+
+def _kind(node):
```

Comment:
It's mostly an abstraction so we don't have to think about how PyTorch stores the operator name and namespace. I suppose we can omit since it's pretty simple.